docker/quickstart/docker-compose.consumers-without-neo4j.quickstart.yml

```diff
@@ -43,10 +41,6 @@ services:
     - ENTITY_REGISTRY_CONFIG_PATH=/datahub/datahub-mce-consumer/resources/entity-registry.yml
     - DATAHUB_SYSTEM_CLIENT_ID=__datahub_system
     - DATAHUB_SYSTEM_CLIENT_SECRET=JohnSnowKnowsNothing
-    - ENTITY_SERVICE_ENABLE_RETENTION=true
```

Comment:
I am concerned about these being removed and the two DATAHUB_* ones above.

Reply:
I can omit the Quickstart generation if needed

Reply:
The issue here is that these env vars were manually put in these files: https://github.com/datahub-project/datahub/pull/6744/files#r1055954698 when they should have been put in non-quick start docker-compose files and these files generated by the `generate_docker_quickstart.sh` file